tests/rest/media/v1/test_media_storage.py

```diff
@@ -535,6 +536,8 @@ def prepare(self, reactor, clock, hs):
         self.download_resource = self.media_repo.children[b"download"]
         self.upload_resource = self.media_repo.children[b"upload"]
 
+        load_legacy_spam_checkers(hs)
```

Comment:
I've fixed this test with the legacy spam checker code so at least this path is evaluated.